.github/workflows/release.yml

```diff
@@ -7,27 +7,119 @@ on:
         uses: docker/login-action@v1
         with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_PASSWORD }}
+      -
+        name: "Docker docker.io Login"
+        uses: docker/login-action@v1
+        with:
+          registry: docker.io
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
-          registry: docker.io
-      - name: Login to quay.io
+      -
+        name: "Build and push ${{ matrix.arch }}"
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./apache.Dockerfile
+          platforms: linux/${{ matrix.arch }}
+          push: true
+          tags: ${{ env.IMAGE }}:${{ github.event.inputs.version }}-${{ matrix.arch }}
+      -
+        id: result
+        name: "Build result outputs version"
+        if: ${{ success() }}
+        run: echo "::set-output name=${{ matrix.arch }}::${{ github.event.inputs.version }}-${{ matrix.arch }}"
+
+  create-manifest:
+    if: always()
+    needs: build-images
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: "Docker quay.io Login"
         uses: docker/login-action@v1
         with:
+          registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-          registry: quay.io
-      - name: Create Release
+      -
+        name: "Create and push manifest"
+        run: |
+          AMEND=""
+          AMD64_VERSION="${{ needs['build-images'].outputs.amd64 }}"
+          if [ -n "$AMD64_VERSION" ]; then
+            AMEND+=" --amend ${{ env.IMAGE }}:$AMD64_VERSION";
+          fi
+          ARM64_VERSION="${{ needs['build-images'].outputs.arm64 }}"
+          if [ -n "$ARM64_VERSION" ]; then
+            AMEND+=" --amend ${{ env.IMAGE }}:$ARM64_VERSION";
+          fi
+          PPC64LE_VERSION="${{ needs['build-images'].outputs.ppc64le }}"
+          if [ -n "$PPC64LE_VERSION" ]; then
+            AMEND+=" --amend ${{ env.IMAGE }}:$PPC64LE_VERSION";
+          fi
+          S390X_VERSION="${{ needs['build-images'].outputs.s390x }}"
+          if [ -n "$S390X_VERSION" ]; then
+            AMEND+=" --amend ${{ env.IMAGE }}:$S390X_VERSION";
+          fi
+          if [ -z "$AMEND" ]; then
+            echo "[!] The job 'build-images' didn't provide any outputs. Can't create the manifest list."
+            exit 1;
+          fi
+          docker manifest create ${{ env.IMAGE }}:${{ github.event.inputs.version }} $AMEND
+          docker manifest push ${{ env.IMAGE }}:${{ github.event.inputs.version }}
+      -
+        id: result
+        name: "Manifest result"
+        if: ${{ success() }}
+        run: echo "Manifest was created and pushed successfully"
+
+  create-release:
+    if: always()
+    needs: create-manifest
+    runs-on: ubuntu-18.04
+    steps:
+      -
+        name: "Checkout Che Dashboard source code"
```

Comment:
I'm sure it doesn't impact much but since you already checked out code on line 30 https://github.com/eclipse/che-dashboard/pull/119/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R30 do you need to do it again?

Reply:
As I have added create-release as a separate job, we need to checkout the code again.